src/components/ContextMenu/ContextMenu.module.scss

```
@@ -0,0 +1,43 @@
+.contextMenu {
+  display: none;
+  position: absolute;
+  top: -0.45em;
+  transform: translateY(-100%);
+  white-space: nowrap;
+  user-select: none;
+  -webkit-user-select: none;
+  -moz-user-select: none;
+  background-color: #c4e3f7;
+  border: 1px solid #09f;
+  border-radius: 2px;
+  -webkit-filter: drop-shadow(0px 0px 0.25em rgba(44, 44, 44, 0.75));
+  filter: drop-shadow(0px 0px 0.25em rgba(44, 44, 44, 0.5));
+  z-index: 2;
+
+  &::before {
+    position: absolute;
+    display: block;
+    content: "";
+    border-style: solid;
+    border-width: 0.6em 0.6em 0 0;
+    border-color: #09f transparent transparent transparent;
+    bottom: -0.6em;
+    left: -1px;
+  }
+
+  &::after {
+    position: absolute;
+    display: block;
+    content: "";
+    border-style: solid;
+    border-width: 0.6em 0.6em 0 0;
+    border-color: #c4e3f7 transparent transparent transparent;
+    bottom: calc(-0.6em + 2px);
+    left: 0;
+    z-index: 1;
+  }
```

We should perhaps explain why `bottom` is the way it is. Maybe a `$border-width` variable might work.

```suggestion
  &::after {
    $border-width: 0.6em;
  
    position: absolute;
    display: block;
    content: "";
    border-style: solid;
    border-width: $border-width $border-width 0 0;
    border-color: #c4e3f7 transparent transparent transparent;
    bottom: calc(-$border-width + 2px);
    left: 0;
    z-index: 1;
  }
```